<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InternetTabsView</class>
 <widget class="QWidget" name="InternetTabsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>660</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="artists">
      <attribute name="title">
       <string>Artists</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="InternetCollectionViewContainer" name="artists_collection" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="albums">
      <attribute name="title">
       <string>Albums</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="InternetCollectionViewContainer" name="albums_collection" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="songs">
      <attribute name="title">
       <string>Songs</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="InternetCollectionViewContainer" name="songs_collection" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="search">
      <attribute name="title">
       <string>Search</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="InternetSearchView" name="search_view" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InternetSearchView</class>
   <extends>QWidget</extends>
   <header>internet/internetsearchview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InternetCollectionViewContainer</class>
   <extends>QWidget</extends>
   <header>internet/internetcollectionviewcontainer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
